spec/lib/traject/config/folio_course_reserves_spec.rb

```diff
@@ -34,4 +34,21 @@
       'EDUC-147-01 -|- Stanford and Its Worlds: 1885-present -|- Mitchell Stevens'
     ]
   end
+
+  context 'for item display' do
```

Comment:
I don't think this is a context (see https://www.betterspecs.org/#contexts).  How about this:
```suggestion
  describe 'the items in the item_display_struct field' do
```

Reply:
I was looking at https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording, but didn't realize that the `for` shown in the example isn't actually a default value.